### PR TITLE
refactor: migrate some components to ts

### PR DESCRIPTION
**Pull request by artemmufazalov**

### CODE REVIEW

File: src/components/Tags/Tags.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import cn from 'bem-cn-lite';
+
+import {Tag, TagType} from '../Tag/Tag';
```
Comment (ariser) [inline src/components/Tags/Tags.tsx:3]: from '../Tag'?

File: src/components/Tags/Tags.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+interface TagsProps {
+    tags: string[];
+    tagsType?: TagType;
+    className?: string;
+}
+
+export const Tags = ({tags, tagsType, className = ''}: TagsProps) => {
+    return (
+        <div className={`${b()} ${className}`}>
```
Comment (ariser) [inline src/components/Tags/Tags.tsx:17]: no need for the `className` default value, and the mix is applied as `b(null, className)` without strings concatenation

File: src/components/EmptyState/EmptyState.tsx [hunk 3]
```diff
@@ -12,7 +13,15 @@ const sizes = {
     l: 350,
 };
 
-export default function EmptyState({image, title, description, actions, size}) {
+interface EmptyStateProps {
+    title: string;
+    image?: ReactNode;
+    description?: string | ReactNode;
```
Comment (ariser) [inline src/components/EmptyState/EmptyState.tsx:19]: ReactNode includes string

File: src/containers/Tablets/Tablets.tsx [hunk 4]
```diff
@@ -104,7 +104,6 @@ export const Tablets = ({path, nodeId, className}: TabletsProps) => {
                 onMouseEnter={onShowTooltip}
                 tablet={tabletsToRender[tabletIndex]}
                 key={tabletIndex}
-                className={b('tablet')}
```
Comment (ariser) [inline src/containers/Tablets/Tablets.tsx:107]: i believe there are styles for this classname in Tablets.scss?

File: src/components/Tablet/Tablet.tsx [hunk 5]
```diff
@@ -0,0 +1,60 @@
+import {useRef} from 'react';
+import cn from 'bem-cn-lite';
+
+import type {TTabletStateInfo} from '../../types/api/tablet';
+
+import {InternalLink} from '../InternalLink';
+
+import {showTooltip} from '../../store/reducers/tooltip';
```
Comment (ariser) [inline src/components/Tablet/Tablet.tsx:8]: oof, don't really like importing a module just for its type
creates an excess edge in dependency graph
but i think this comment is just to bring attention to it, not to fix right away

**Review by ariser — CHANGES_REQUESTED** [review]

File: src/containers/Tablets/Tablets.tsx (hunk 4 above)
Comment (artemmufazalov) [reply src/containers/Tablets/Tablets.tsx:107]: These styles wasn't applied in `Tablet`. So I removed prop but forgot to remove styles themselves.

**Review by ariser — APPROVED** [review]

**Pull request merged**